web/index.html

```diff
@@ -1435,18 +1435,29 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 		colors: '#FF5588',
 		heads: [
 			gaugeChart('Utilization', '12%', '', '#FF5588')
-		]
+		],
+		info: 'Percent of time the disk was doing something. Read from field 10.'
 	},
 
 	'disk.backlog': {
-		colors: '#0099CC'
+		colors: '#0099CC',
+		info: 'Milliseconds spent doing I/Os multiplied by the number of operations currently in progress. Read from field 11.'
```

Comment:
Here I kept the text short because:
- The value is no accurate backlog.
- For disks which are able to do operations in parallel this value is a multiple of the "real" backlog.

I suggest this text:

Backlog is an indication of the duration of pending disk operations. On every I/O event the Linux kernel is multiplying the time spent doing I/O since the last update of this field with the number of pending operations. While not accurate, this metric can provide an indication of the expected completion time of the operations in progress.